### PR TITLE
Fixes the IConvertible warning thrown on Windows Phone reported on #317

### PR DESCRIPTION
**Pull request by MehdiK**

### CODE REVIEW

**Comment by MehdiK** [comment]
@PombeirP can you please clone this branch, build and reference it in your project to see if it removes [your last warning](https://github.com/MehdiK/Humanizer/issues/317#issuecomment-72118928)? If you can verify this, I will then merge and release this. Thanks.

**Comment by PombeirP** [comment]
@MehdiK I've tested it and it fixed the warning. Thanks!

File: src/Humanizer.Tests/DateHumanizeDefaultStrategyTests.cs [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void HoursFromNowNotTomorrow(int hours, string expected)
         [Theory]
         [InlineData(1, "yesterday")]
         [InlineData(10, "10 days ago")]
-        [InlineData(28, "28 days ago")]
+        [InlineData(27, "27 days ago")]
```
Comment (mexx) [inline src/Humanizer.Tests/DateHumanizeDefaultStrategyTests.cs]: Why this change? Oh, let me guess cause it's February?

Comment (MehdiK) [inline src/Humanizer.Tests/DateHumanizeDefaultStrategyTests.cs]: Thanks for the review. Yes, you're right. Suddenly the unit tests started failing!! Do'h!

**Comment by MehdiK** [comment]
Thanks for checking @PombeirP. This is now released to NuGet. Please give it a try and see if you're all green!

**Comment by PombeirP** [comment]
@MehdiK Confirmed. Thanks for working through this with me, got it compiling again out-of-the-box with no warnings. Awesome!

**Comment by MehdiK** [comment]
Awesome. That took quite a few shots but I'm glad it's finally sorted. Thanks for your patience and help on this.
